Program.cs

```diff
@@ -358,7 +400,24 @@ private static async Task<List<MergeBaseCommit>> LoadCommitsForReleasesAsync(str
 
                 return releaseCommits;
             }
-            catch (System.Exception ex)
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+
+            return null;
+        }
+
+        private static List<dotnetthanks.Release> LoadCurrentCoreJson()
+        {
+            try
+            {
+                string fileName = "core.json";
```

Comment:
if the file doesn't exist, it should create an empty one. Right now, it just fails with a filenotfound exception.